yb-voyager/cmd/importDataFileTaskImporter.go

```diff
@@ -246,6 +271,17 @@ func getImportBatchArgsProto(tableNameTup sqlname.NameTuple, filePath string) *t
 	if err != nil {
 		utils.ErrExit("if required quote column names: %s", err)
 	}
+
+	// TODO: Implement caching since this is called for every batch
```

Comment:
we might have to do this as part of this overall task before release. catalog queries on YB are quite slow, so I suspect perf degradation because of this.